ui/components/app/transaction-list-item-details/index.scss

```diff
@@ -62,9 +64,10 @@
   }
 
   &__operations {
-    margin: 0 0 16px 16px;
+    margin: 0 16px 16px 16px;
     display: flex;
-    justify-content: flex-end;
+    justify-content: space-between;
```

Comment:
When there's a retry button, we need to handle the alignment as well 
<img width="356" alt="Screenshot 2024-12-19 at 03 29 53" src="https://github.com/user-attachments/assets/a206c3d4-e2b4-420e-84b6-e9ab9ea949e2" />